### ISSUE TITLE
console error only print {}, without effect message.

### ISSUE THREAD

**Issue by JFRabbit** [archfz/cypress-terminal-report#41]
support/index.js
```
collectTypes: [
          "cons:error",
          "cy:log",
          "cy:xhr",
          "cy:request",
          "cy:route",
          "cy:command",
        ],
```

<img width="369" alt="截屏2020-08-13 下午9 08 54" src="https://user-images.githubusercontent.com/22906174/90138279-5d22e180-dda9-11ea-87bb-aafd5b42e16e.png">
<img width="382" alt="截屏2020-08-13 下午9 09 01" src="https://user-images.githubusercontent.com/22906174/90138282-5eeca500-dda9-11ea-99f9-83efa4c60808.png">

**Comment by JFRabbit** [comment]
This message should be a catched error from app, maybe like this code:
``` js
var a = 1;
try {
  a.split(",");
} catch (e) {
  // cypress-terminal-report print error line in terminal, but show "cons:error ✘  {}"
  // in browser console, I can get "TypeError: a.split is not a function
  console.error(e); 
}
a.split(","); // make cypress run failed when "uncaught:exception" return true
```

<img width="865" alt="截屏2020-08-19 下午3 11 26" src="https://user-images.githubusercontent.com/22906174/90603959-b027e900-e22e-11ea-9ebd-776a56e8c31a.png">
<img width="761" alt="截屏2020-08-19 下午3 12 12" src="https://user-images.githubusercontent.com/22906174/90603971-b4540680-e22e-11ea-9745-c3829005207d.png">

**Comment by JFRabbit** [comment]
looks like ```console.error(error)``` will print ```{}```, use string is ok.

**Comment by archfz** [comment]
I tried to reproduce the issue you have with the same code but I couldnt. The issue is that for some reason on your code the error that is thrown is not instance of window.Error. Another possibility is that the error doesnt have the stack property but I doubt that. I have an idea for a possible fix, I will release next week in a patch so we can try if it resolves it for you.

In the meantime:
* what browser do you use?
* do you have any polyfills that might change the error object?
* can you check in the catch whether error instanceof window.Error ?

**Comment by JFRabbit** [comment]
- what browser do you use?

Browser:    Electron 80

- can you check in the catch whether error instanceof window.Error ?

Yes

You create a react app to reproduce this issue:

``` js
import React, { Component } from "react";
import { Button } from "antd";
import "antd/dist/antd.css";

class App extends Component {
  makeTypeError = () => {
    var a = 1;
    try {
      a.split(",");
    } catch (e) {
      console.error(e);
      console.info(e instanceof window.Error);
    }
    a.split(",");
  };

  render() {
    return (
      <div>
        <Button id="error" onClick={this.makeTypeError}>
          Make Type Error
        </Button>
      </div>
    );
  }
}

export default App;

```

test code:
```js
describe("foobar", () => {
  it("foobar", () => {
    cy.visit("http://localhost:3000/");

    cy.get("#error").click();
  });
});

```

Terminal:
``` log
      cy:command ✔  visit       http://localhost:3000/
      cy:command ✔  get #error
      cy:command ✘  click
          cy:xhr ⓘ  GET http://localhost:3000/sockjs-node/info?t=1598261034294
      cons:error ✘  {}
```

**Comment by archfz** [comment]
Released in 2.0.0.
